scripts/update-component-sha.sh

```diff
@@ -1,10 +1,66 @@
 #!/bin/sh
 set -e
-if [ $# -ne 2 ] ; then
-    echo "Need <OLD> and <NEW> as arguments" >&2
+
+##
+#
+# script to replace hashes in config files
```

Comment:
Heh, because this was my usage guide before I wrote `usage`! First I doc how it will work, then write it, then realized we needed better `usage` so added it. :-)